### PR TITLE
Move examples over

### PR DESCRIPTION
**Pull request by alexrockhill**
Fixes a couple items in the checklist for moving examples over here https://github.com/mne-tools/mne-gui-addons/issues/5

### CODE REVIEW

**Comment by alexrockhill** [comment]
Good to go, I think @larsoner once this merges I can link them in mne-python.

I don't have write access so I can't merge also.

**Comment by alexrockhill** [comment]
Well actually everything worked except the 3D brain plots didn't render in sphinx: https://output.circle-artifacts.com/output/job/70b0c79d-f6f7-44ac-9dd8-ae684053bb09/artifacts/0/dev/auto_examples/ieeg_locate.html. I'm not sure what setting that is that isn't copied over...

File: .circleci/config.yml [hunk 1]
```diff
@@ -30,6 +30,7 @@ jobs:
               echo "set -e" >> $BASH_ENV
               echo "export OPENBLAS_NUM_THREADS=4" >> $BASH_ENV
               echo "export XDG_RUNTIME_DIR=/tmp/runtime-circleci" >> $BASH_ENV
+              echo "export MNE_3D_OPTION_MULTI_SAMPLES=1" >> $BASH_ENV
```
Comment (larsoner) [inline .circleci/config.yml:33]: This was the magic. Our detection of MESA in MNE-Python must be flawed...

Comment (alexrockhill) [reply .circleci/config.yml:33]: Good to know, thanks for fixing that

**Pull request merged**